packages/dai-plugin-migrations/contracts/abiMap.json

```diff
@@ -1,3 +1,4 @@
 {
-  "TUB": "SaiTub"
+  "TUB": "SaiTub",
+  "REDEEMER": "Redeemer" 
```

Comment:
The redeemer contract is deployed to the testchain properly now, but won't be used until a subsequent task implementing the migration